build.gradle

```diff
@@ -46,7 +46,8 @@ subprojects {
         myriadExecutorConf
         myriadExecutorConf.transitive = false
         // exclude hadoop/yarn deps for 'runtime'
-        runtime.exclude group: 'org.apache.hadoop', module: '*'
+        // TODO (Kannan Rajah) Exclude only while packaging
+        //runtime.exclude group: 'org.apache.hadoop', module: '*'
```

Comment:
Agreed. This is on my list. The unit tests @rkannan82 added didn't seem to run unless this is commented. Need to do some gradle magic to make unit tests run and yet keep the jars from being put in ```myriad-scheduler/build/libs/``` during production build.